app/src/main/java/com/concordium/wallet/ui/passphrase/recover/RecoverWalletActivity.kt

```diff
@@ -78,11 +85,25 @@ class RecoverWalletActivity : BaseActivity() {
     }
 
     private fun initObservers() {
-        viewModel.validate.observe(this) { success ->
-            if (success)
+        viewModel.seed.observe(this){seed ->
+                showAuthentication(this) { password ->
+                    password?.let {
+                            viewModel.setSeedPhrase(seed, password)
+                    }
+                }
+        }
+        viewModel.saveSeed.observe(this){
```

Comment:
I think the parameter should be named `saveSuccess` like it is above.

Reply:
Done